### PR TITLE
[Pubsub] Add pubsub module stats

### PR DESCRIPTION
**Pull request by rkooo567**
## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by clarkzinzow — APPROVED** [review]
Nice! I'm assuming that there are some future plans to add publisher-side stats as well? And it might be nice to have a running sum for the number of long-poll responses, and maybe the number of active long-polls as well?

File: src/ray/pubsub/subscriber.h [hunk 1]
```diff
@@ -290,6 +302,8 @@ class Subscriber : public SubscriberInterface {
     return it->second;
   }
 
+  const std::string DebugString() const;
```
Comment (iycheng) [inline src/ray/pubsub/subscriber.h:305]: The return doesn't need to be const.

**Comment by iycheng** [comment]
LGTM, one comment.

**Comment by rkooo567** [comment]
@clarkzinzow Yes you are right! Right now, we don't periodically print DebugString from core worker, so I didn't add that to the publisher, but I will potentially add it in the future. Btw I will close this PR and pick this up to https://github.com/ray-project/ray/pull/16487

**Pull request closed without merging**